src/styles/Layout.module.css

```diff
@@ -1,30 +1,60 @@
-* {
-  box-sizing: border-box;
-}
 
 .layout {
+  background: white;
+  margin-top: 0;
+  height: 12vh;
   display: flex;
   justify-content: space-between;
   align-items: center;
-  width: 100%;
+  padding-left: 50px;
+  padding-right: 50px;
+  box-shadow: 2px 2px 2px 1px rgba(0, 0, 0, 0.1);
 }
 
 .container {
+  width: 100%;
+  background: #f5f6fa;
+}
+
+.navigation {
```

Comment:
- [ ] Kindlynote that the styling  should be  **EXACTLY** the same as the one provided on [Zeplin](https://zeplin.io/)Styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). you need to use the same layout, font-size dimentions, positions of items, colors, ,margins, paddings...

> Your design 
>  ![image](https://github.com/Samuelwanza/bookstore/assets/65068771/25fadf8a-5cf1-4b6c-9a7c-41b141a6eaf3)
>The template
> ![image](https://github.com/Samuelwanza/bookstore/assets/65068771/7abafbac-51c7-45c9-aeeb-636c2f0ad903)

Reply:
- [ ] Like the previous reviewer mentioned, please make sure the design is the same as that seen in the original design on Zeplin. 
  * Please add the line above the "ADD NEW BOOK" form.
  *  Please make sure the "ADD BOOK"  button is aligned with the cards containing the books' information.
  * The input fields should be closer to each other as seen in the original design
![image](https://github.com/Samuelwanza/bookstore/assets/45460192/d0bd12d1-8bdf-49bc-b81d-1255df5ecad3)